numpy/core/src/_simd/_simd_inc.h.src

```diff
@@ -147,22 +147,40 @@ typedef struct
 {
     PyObject_HEAD
     // vector type id
-    simd_data_type type;
+    simd_data_type dtype;
     // vector data, aligned for safe casting
     npyv_lanetype_u8 NPY_DECL_ALIGNED(NPY_SIMD_WIDTH) data[NPY_SIMD_WIDTH];
-} simd_vector;
+} simd_vector_obj;
 /**
- * convert simd_data to PyObject(simd_vector),
- * raise Python exception on failure and returns NULL.
+ * Create a Python obj(simd_vector_obj) from a NPYV vector based on the contents
+ * of `data`(simd_data) and according to the vector data type `dtype`
+ * on range(simd_data_[vu8:vf64]).
+ * Return NULL and a Python exception on failure, otherwise new reference.
+ *
+ * Example:
+ ** simd_data data = {.vu8 = npyv_setall_u8(0xff)};
+ ** simd_vector_obj *obj = simd_vector_to_obj(data, simd_data_vu8);
+ ** if (obj != NULL) {
+ **    printf("I have a valid vector obj and first element is \n", obj->data[0]);
+ **    Py_DECREF(obj);
+ ** }
  */
-static simd_vector *
-simd_vector_to_obj(simd_data data, simd_data_type vtype);
+static simd_vector_obj *
+simd_vector_to_obj(simd_data data, simd_data_type dtype);
```

Comment:
I think given your comment is "create a ... simd_vector_obj from ...", it's hard to argue that this function should be called `to_obj` and not `from_...`